domains/temelkov

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Stanislav Temelkov Professional Portfolio",
+  "repo": "https://github.com/Temelkov/TEMELKOV.IS-A.DEV",
+  "owner": {
+    "username": "temelkov",
+    "email": "stanislav.temelkov@hotmail.com"
+  },
+  "record": {
+    "CNAME": "temelkov-tk-2022.vercel.app"
```

Comment:
Vercel is not supported yet (might be soon)

Reply:
Okey, do this with this CNAME: https://temelkov.github.io/TEMELKOV-IS-A-DEV/

Reply:
@Stef-00012